### PR TITLE
Add support for authenticated fees endpoint

### PR DESCRIPTION
**Pull request by DoWhileGeek**
I wanted to spam this endpoint, and noticed it wasnt supported on this library, only took a few minutes to do with this super clean lib to copypasta from.

### CODE REVIEW

File: tests/authenticated.spec.js [hunk 1]
```diff
@@ -609,6 +609,35 @@ suite('AuthenticatedClient', () => {
...
+      authClient.getFundings((err, resp, data) => {
+        if (err) {
+          reject(err);
+        }
+        assert.deepEqual(data, expectedResponse);
+        resolve();
+      });
+    });
+
+    let promisetest = authClient.getFundings();
```
Comment (vansergen) [inline tests/authenticated.spec.js]: `getFundings` should be replaced with `getFees `

File: tests/authenticated.spec.js [hunk 2]
```diff
@@ -609,6 +609,35 @@ suite('AuthenticatedClient', () => {
...
+      usd_volume: '500.00',
+    };
+
+    nock(EXCHANGE_API_URL)
+      .get('/funding')
+      .times(2)
+      .reply(200, expectedResponse);
+
+    let cbtest = new Promise((resolve, reject) => {
+      authClient.getFundings((err, resp, data) => {
```
Comment (vansergen) [inline tests/authenticated.spec.js]: `getFundings` should be replaced with `getFees `

File: tests/authenticated.spec.js [hunk 3]
```diff
@@ -609,6 +609,35 @@ suite('AuthenticatedClient', () => {
...
 
+  test('.getFees()', done => {
+    const expectedResponse = {
+      maker_fee_rate: '0.0050',
+      taker_fee_rate: '0.0050',
+      usd_volume: '500.00',
+    };
+
+    nock(EXCHANGE_API_URL)
+      .get('/funding')
```
Comment (vansergen) [inline tests/authenticated.spec.js]: `funding` should be replaced with `fees`

**Review by vansergen — COMMENTED** [review]
It might be a good idea to add an example to the `README`.

File: tests/authenticated.spec.js (hunk 1 above)
Comment (DoWhileGeek) [reply tests/authenticated.spec.js]: Oh gosh, sorry, I  saw your comments and couldve swore I swapped everything out with `fees`. So Iwent to my editor, and noticed I didnt save. Sorry about that! Should be all fixed now.

**Comment by DoWhileGeek** [comment]
@vansergen not to be pushy, but this pr is ready for another look.

**Comment by drewrothstein** [comment]
Hi, we are closing out PRs + Issues as this project is being archived.

**Pull request closed without merging**